### PR TITLE
Fix display of negative numbers in weekly digest

### PR DESCRIPTION
**Pull request by ezraberch**
Old:
```
· Generated projects: 53254 (-8% less than the week before, 58064)
```
New:
```
· Generated projects: 53254 (8% less than the week before, 58064)
```

### CODE REVIEW

**Review by danyf90 — APPROVED** [review]

File: projects/backbone/app/jobs/weekly_digest_job.rb [hunk 1]
```diff
@@ -71,7 +71,7 @@ def perform(channel: "#general")
 
   def more_or_less_percentage_in_words(percentage)
     if percentage < 0
-      "#{percentage}% less than"
+      "#{-percentage}% less than"
     else
       "#{percentage}% more than"
```
Comment (danyf90) [inline projects/backbone/app/jobs/weekly_digest_job.rb:76]: Maybe writing it differently it's more evident if things are going down or up, something like:
```
  def more_or_less_percentage_in_words(percentage)
    "#{percentage}% with respect to"
  end
```

Comment (danyf90) [reply projects/backbone/app/jobs/weekly_digest_job.rb:76]: We could even go fancier and add some emoji like ⬆️ /📈 and ⬇️ /📉

**Review by fortmarek — APPROVED** [review]

File: projects/backbone/app/jobs/weekly_digest_job.rb (hunk 1 above)
Comment (fortmarek) [reply projects/backbone/app/jobs/weekly_digest_job.rb:76]: Love the idea of emojis!

**Comment by danyf90** [comment]
We can revisit the messages in a separate PR, as I think we also need to change some logic after 3.0 release 🤔

**Pull request merged**